README.md

```diff
@@ -1,33 +1,71 @@
- 
+
 ## Stakeholders
 In 2015, the Community Resource Team (CRT) developed out of a 3-year U.S. Department of Justice Byrne Criminal Justice Innovation (BCJI) grant and is a cross-sector collaboration of community leaders, social services providers, government institutions and law enforcement that jointly identify and address community safety issues in the Prospect Avenue corridor. CRT works to create a safer community and build collective efficacy within the 2.15 square mile area from Paseo Boulevard to Indiana Avenue and from 25th Street to 39th Streets, an area identified because of the prevalence of violent crime.
- 
+
 The CRT Graffiti Abatement Pilot Project area will stretch from 18th Street to 39th, 3-4 blocks east and west of Prospect Avenue. This project will indirectly serve residents and business owners in the area.
- 
+
 ## Approach / Roadmap:
- 
+
 
 ## Team:
 * Amanda Wilson
 * Jake LaCombe
 * Aaron McRuer
 * Kol Kheang
+* Fred Lawl
 
 Code for KC Slack Channel: #taggingtracker
 
 
 ## Developers Notes
 
+### Pre-requisites Tools
+
+- [Xcode & Xcode Command Line Tool](https://developer.apple.com/xcode/)
+- [Homebrew](https://brew.sh/)
+- [Node.js](https://nodejs.org/en/download/)
+
+### Dev Environment Config -- dev.env
+
+The project requires an environment file to run successfully.  For `dev` environment, `dev.env` file must be created in the project directory with the following environment variables:
+
+```
+AUTH0_CLIENTID - ClientID of the auth0 authentication engine
+AUTH0_DOMAIN - FQDN of the auth0 authentication engine
+TAGGING_TRACKER_SERVICE_DOMAIN - http://localhost/ when working locally or change remote host and port otherwise
+```
+
+When connecting to Azure cloud, the following environment variables are required:
+
+```
+AZURE_CONTAINER_KEY - Azure container key ID
+AZURE_IMAGE_CONTAINER_NAME - Azure image container name
+AZURE_IMAGE_CONTAINER_KEY - Azzure image container key
+```
+
+Contact one of the project owners to get a pre-populated `dev.env` file with all the environment variables. You can also register a new account @ auth0.com.
+
+### Dependencies
+
+- Related backend project: [tagging_tracker_backend](https://github.com/codeforkansascity/tagging_tracker_backend)
+- Auth0 service: [App Authentication](https://auth0.com/)
+
```

Comment:
I would move dependencies to close to or before the "Pre-requisite Tools" because this project has a dependency on these apps.